index.html

```diff
@@ -13,8 +13,9 @@ <h3>Talk to your bot!</h3>
       <h3>Chat history</h3>
       <div id="chatBot">
           <textarea id="output" name="ChatHistory" rows="15"></textarea>
-      </div>  
+      </div>
   </div>
+  <script src="chatbox.js" defer></script>
```

Comment:
Since you've include the script at the end before the closing body tag, you don't need to use the `defer` keyword here because at the point your script is encountered, the HTML of your page will have been parsed and loaded already. You'd use the `defer` keyword if you were to include your `<script>` tag in the head or before other HTML elements that your script would be working with. [More info here](https://javascript.info/script-async-defer).